meta/main.yml

```diff
@@ -14,7 +14,7 @@ galaxy_info:
     - ptp
   company: Red Hat, Inc.
   license: MIT
-  min_ansible_version: 2.4
+  min_ansible_version: 2.9
```

Comment:
Is there anything that requires Ansible 2.9? 2.7 and 2.8 are still supported, we need to support them.

Reply:
I don't believe this will be a problem, I just am currently using 2.9, and didn't want to jump the gun on saying it works.